tests/integ/test_clarify.py

```diff
@@ -201,6 +483,34 @@ def test_pre_training_bias(clarify_processor, data_config, data_bias_config, sag
         check_analysis_config(data_config, sagemaker_session, "pre_training_bias")
 
 
+def test_pre_training_bias_facets_not_included(
+    clarify_processor, data_config_facets_not_included, data_bias_config, sagemaker_session
+):
+    with timeout.timeout(minutes=CLARIFY_DEFAULT_TIMEOUT_MINUTES):
+        clarify_processor.run_pre_training_bias(
+            data_config_facets_not_included,
+            data_bias_config,
+            job_name=utils.unique_name_from_base("clarify-pretraining-bias-facets-not-included"),
+            wait=True,
+        )
+        analysis_result_json = s3.S3Downloader.read_file(
+            data_config_facets_not_included.s3_output_path + "/analysis.json",
+            sagemaker_session,
+        )
+        analysis_result = json.loads(analysis_result_json)
+        assert (
+            math.fabs(
+                analysis_result["pre_training_bias_metrics"]["facets"]["F1"][0]["metrics"][0][
+                    "value"
+                ]
+            )
+            <= 1.0
```

Comment:
If you set the seed in `SHAPConfig`, you might be able to assert exact values.

Reply:
We're not running SHAP or PDP, and the parameters of `run_pretraining_bias` don't currently support an input `SHAPConfig`, so I don't think that solution would work here.